samples/nextjs/sitecore/config/JssNextWeb.config

```diff
@@ -18,6 +18,16 @@
         thus making analytics track the correct original client IP address.
       -->
       <setting name="Analytics.ForwardedRequestHttpHeader" set:value="X-Forwarded-For" />
+
+      <!--
+        JSS EDITING SECRET
+        To secure the Experience Editor endpoint exposed by your Next.js app (see `serverSideRenderingEngineEndpointUrl` below),
+        a secret token is used. This is taken from an env variable by default, but could be patched and set directly by uncommenting.
+        This (server-side) value must match your client-side value, which is configured by the JSS_EDITING_SECRET env variable (see .env file).
```

Comment:
> see .env file

This could be confused for the docker-compose `.env`

Reply:
Added clarification "see the Next.js .env file"